Dockerfile

```diff
@@ -76,6 +82,20 @@ ARG COMMIT_SHA
 ENV SHA=${COMMIT_SHA}
 RUN echo ${SHA} > public/check
 
+# Create writable directories for Chrome
+RUN mkdir -p /app/tmp /home/appuser && \
+    chmod 755 /app/tmp
+
+# Change ownership of app directory and home directory for write access
+RUN chown -R appuser:appgroup /app && \
```

Comment:
we want to run without recursive chown of /app 
if we have to chown selected directories like /app/log or /app/tmp (as above) then fine